srcpkgs/yacy/template

```diff
@@ -0,0 +1,30 @@
+# Template file for 'yacy'
+pkgname=yacy
+version=1.922
+revision=1
+archs="x86_64"
+depends="openjdk-jre"
+short_desc="YaCy p2p decentralized search engine"
+maintainer="Jason Manley <jason@jasondavid.tv>"
+license="GPL-3.0-or-later"
+homepage="https://yacy.net"
+distfiles="http://latest.yacy.net/yacy_v${version}_20191013_9964.tar.gz"
+checksum=@99af963642dafaeab2657ad59315e7b7896bf4c1e7091f8f67e95e71802d44df
+
+system_accounts="yacy"
+yacy_homedir="/opt/yacy"
+yacy_shell="/bin/bash"
+make_dirs="/opt/yacy 0755 yacy yacy"
+
+post_extract() {
+mv ${XBPS_BUILDDIR}/{yacy,yacy-1.922}
+}
+
+do_install() {
+vmkdir opt/yacy
+vcopy $XBPS_BUILDDIR/yacy-1.922/* opt/yacy
+}
+
+post_install() {
+vsv yacy
+}
```

Comment:
indent with tabs

Reply:
It works perfectly well in this case of not building from source. I removed the arch restriction too. I did do some testing with "ant clean all" in do_install but can't figure out what java package is needed in makedepends. It keeps erroring out trying to find java.

Reply:
You need the actual development kit to build something with java, so add `openjdk` to `hostmakedepends`. You might need to set `JAVA_HOME` as well.

Reply:
I did try with openjdk as a makedepends (in addition of course to apache-ant), but I'm not too familiar with how JAVA_HOME should be set here.

Reply:
Take a look at how lightzone does it, for example.